mlir/include/mlir/Dialect/Transform/IR/TransformOps.td

```diff
@@ -614,12 +618,14 @@ def GetParentOp : TransformDialectOp<"get_parent_op",
     is applied, e.g., "B" may itself be a parent of "A". This may have an impact
     on the further transformation applied to the handle produced here.
 
-    If any of the given Payload IR ops has no such suitable parent, the
-    transformation fails silently.
+    If any of the given Payload IR ops has no such suitable parent, then:
+      - if `allow_empty_results` is set, the result handle is empty
+      - otherwise, the transformation fails silently.
```

Comment:
Let's use "produces a silenceable failure" instead. "Fails silently" sounds like it could continue executing. The failure can be silenced, but is not silent. By default, it will be reported.